static/css/theme-osum.css

```
@@ -196,6 +196,21 @@ tr:last-child td:last-child {
   border-bottom-right-radius: 4px;
 }
 
+
+sub, sup {
+  font-weight: bold;
+  vertical-align: top;
+}
+
+sup {
+  top: unset;
+}
+
+h1 sub,
+h1 sup {
+  font-size: 0.6em;
+}
+
```

Looks a bit weird on `<h6>`, but who uses `<h6>` titles... I'd add this. 

```suggestion

h6 sub,
h6 sup {
  font-size: 0.5em;
}

```

---

I know but this is an issue with the headings' style. I'll open a follow up bug.